.github/workflows/merge.yml

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: Apache-2.0
+
+name: Merge
+
+on:
+  push:
```

Comment:
By the time the commit is pushed isn't it too late to do anything about it?

Reply:
No, because we've enabled [bors](https://bors.tech/) in this repo, the pre-merge (on staging branch) check will fail and bors requires that [check to pass](https://github.com/substrait-io/substrait-rs/pull/48/files#diff-71db1a12d4fd6edf3691ae58f7a7ef000e29931f086ee79ff328f32e9e66a4c0R10), so it will cancel the merge and comment on the PR that that check failed.